client/modules/Booking/Booking.js

```diff
@@ -50,40 +51,40 @@ export class Booking extends Component {
 				startDate: false,
 				endDate: false,
 			}, 
-		  };
-		  this.isFormInvalid = this.isFormInvalid.bind(this);
-	
+		};
+		this.isFormInvalid = this.isFormInvalid.bind(this);
+
 	}
 //Set up variables for Msgs
 
 labels = {
 	startDate: 'Start Time',
 	endDate: 'End Time',
-  };
+};
 
-  errorMsgs = {
+errorMsgs = {
 	startDate: 'a valid date and time at least '+2*this.intervalNum+' '+this.intervalUnit+' in the future is required',
 	endDate: 'a date and time after start time is required',
-  };
-  
-	componentDidMount() {
-		if(window.localStorage.getItem('JWT'))
-		{
-			this.setState({	loggedIn:true });
-			this.state.userid=JSON.parse(atob(window.localStorage.getItem('JWT').split('.')[1]))['sub'];
-		}
-		this.setState({	carid:this.props.location.query.carid });
-		http.client()
-		.get("/cars")
-      	.then(res => {
-        	this.mapCarToModel(res.data);
-      	})
-      	.catch(err => {
-        	console.log(err);
-		});
-	}	
+};
+
+componentDidMount() {
+	if(window.localStorage.getItem('JWT'))
+	{
+		this.setState({	loggedIn:true });
+		this.state.userid=JSON.parse(atob(window.localStorage.getItem('JWT').split('.')[1]))['sub'];
```

Comment:
should `this.state.userid=` be in a `this.setState`?
and
if the prev comment re `const token = storage.get(storage.Keys.JWT);` gets implemented then userid = JSON.parse(atob(token.split('.')[1]))['sub'];